Makefile

```diff
@@ -0,0 +1,15 @@
+NODE_BIN=./node_modules/.bin
+
+requirements:
+	npm install
+
+develop: requirements
+
+test: requirements
+	$(NODE_BIN)/karma start --single-run
+
+test-dev: requirements
+	$(NODE_BIN)/karma start
+
+quality: requirements
+	$(NODE_BIN)/jshint lib/ test/
```

Comment:
By the way, we have .jshintrc files in some repos, like [here](https://github.com/edx/edx-analytics-dashboard/blob/master/.jshintrc). I can't remember, but this might be a standard assumption and therefore worthwhile including here. Could you check with Andy and Dennis?